src/flint/fq.jl

```diff
@@ -581,25 +581,27 @@ function FlintFiniteField(char::Integer, deg::Int, s::AbstractString; cached = t
 end
 
 @doc Markdown.doc"""
-    FlintFiniteField(pol::fmpz_mod_poly, s::AbstractString; cached = true)
+    FlintFiniteField(pol::PolyElem, s::AbstractString; cached = true, check = true)
```

Comment:
I don't understand this change. It almost certainly won't work.

Reply:
I suppose it was to provide a more general documentation, as no docstring was attached to the version taking an `gfp_fmpz_poly`. But I agree `PolyElem` is too vague. I will add a note in the docstring specifying that this is currently only supported for `fmpz_mod_poly` and `gfp_fmpz_poly`.

Reply:
Ok, I removed this change because I can't build the documentation locally and test it. I will try to fix the failing doc build in another PR.